modules/manage/partials/authentication.adoc

```diff
@@ -0,0 +1,1661 @@
+
+[[scram]]
+==== SASL/SCRAM
+
+SASL/SCRAM does not require sending passwords over the network, even in an encrypted form. It uses a challenge-response mechanism, ensuring that the password is not directly accessible to the server. It works with hashed passwords, providing additional security against dictionary attacks.
+
+ifndef::env-kubernetes[]
+===== Enable SASL/SCRAM
+
+SASL/SCRAM is enabled by default.
+
+To check if SASL/SCRAM is enabled:
+
+[,bash]
+----
+rpk cluster config get sasl_mechanisms
+----
+
+You should see `SCRAM` in the output.
+
+If SASL/SCRAM is not enabled, enable it:
+
+[,bash]
+----
+rpk cluster config set sasl_mechanisms '["SCRAM"]'
+----
+endif::[]
+
+===== Create SCRAM users
+
+When you have SASL authentication enabled for your Redpanda cluster, you can create SCRAM users. Redpanda supports the following SASL/SCRAM authentication mechanisms for the Kafka API:
+
+- `SCRAM-SHA-256`
+- `SCRAM-SHA-512`
+
+By default, SCRAM users don't have any permissions in the cluster. Only superusers can grant permissions to new users through ACLs.
+
+. To create the user `myuser` with a password `changethispassword`, run xref:reference:rpk/rpk-acl/rpk-acl-user-create.adoc[`rpk acl user create`]:
++
+```bash
+rpk acl user create myuser \
+  -p 'changethispassword' \
+  --mechanism SCRAM-SHA-256
+```
++
+TIP: Enclose passwords in single quotes to avoid conflicts with special characters. Enclosing characters in single quotes preserves the literal value of each character.
+
+. Use the xref:reference:rpk/rpk-acl/rpk-acl-create.adoc[`rpk acl create`] command to grant `create` and `describe` permissions to `myuser` in the cluster:
++
+```bash
+rpk acl create --allow-principal User:myuser \
+  --operation create,describe \
+  --cluster \
+  -X user=<superuser-name> \
+  -X pass='<superuser-password>' \
+  -X sasl.mechanism=<superuser-authentication-mechanism>
+```
+
+. Grant the new user `describe` privileges for a topic called `myfirsttopic`:
++
+```bash
+rpk acl create --allow-principal User:myuser \
+  --operation describe \
+  --topic myfirsttopic \
+  -X user=<superuser-name> \
+  -X pass='<superuser-password>' \
+  -X sasl.mechanism=<superuser-authentication-mechanism>
+```
++
+NOTE: You must grant privileges for specific topics. Even if a user has `describe` privileges for a cluster, it does not mean that the user is granted `describe` privileges for topics.
+
+See also: xref:manage:security/authorization.adoc#user-create[User create].
+
+===== Connect to Redpanda
+
+This section provides examples of connecting to Redpanda as a SCRAM user when SASL/SCRAM authentication is enabled.
+
+Create a topic as the `myuser` user by running xref:reference:rpk/rpk-topic/rpk-topic-create.adoc[`rpk topic create`]:
+
+```bash
+rpk topic create myfirsttopic \
+  -X user=myuser \
+  -X pass='changethispassword' \
+  -X sasl.mechanism=SCRAM-SHA-256
+```
+
+[TIP]
+====
+You can also use environment variables to set the user credentials instead of flags:
```

Comment:
I suspect that profiles are recommended now, although ENV is also useful. Not sure these alternative configuration mechanisms aren't more noise than signal in this part of the doc.